src/main/java/org/example/exception/SqlException.java

```diff
@@ -0,0 +1,7 @@
+package org.example.exception;
+
+public class SqlException extends JobRoleDaoException {
```

Comment:
I am confused what this is for?

Reply:
That was a mistake in my understanding, it doesn't need to exist since we should handle the SQL exception in the dao, removed